include/DetectorModule.hh

```diff
@@ -247,6 +257,8 @@ public:
   const bool isAtPlusXSide() const { return (center().X() >= -insur::geom_zero); }
   double tiltAngle() const { return tiltAngle_; }
   bool isTilted() const { return tiltAngle_ != 0.; }
+  double twistAngle() const { return twistAngle_; }
+  const XYZVector& getRAxis() const {return rAxis_;}
```

Comment:
Is this the centre of the sensor? Again, I thought this vector is already available.

Reply:
Would be nicer to call it yawAngle (not important, but that makes the code more readable eventually because one can see out of context the type of rotation the variable refers to)

Reply:
`getRAxis()` is probably actually not needed here.

RE the naming: happy to go with whatever the consensus is here.

Reply:
Yes basically twist makes more think of the deformation of a solid (a torsion?) and is not transparent on the rotation axis.
In the private branch I had done I had used yawAngle (https://en.wikipedia.org/wiki/Yaw_(rotation)), I would suggest that, but as you want.